wait-for-it.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+until ping -c 1 postgres >/dev/null 2>&1; do echo "not up"; done
+
+if [ "$NEW_BUILD" = true ]; then
+    npm run create-history-db 
```

Comment:
runs the create-history commands if the NEW_BUILD env var is present on the ntt container in docker-compose